consensus/core/src/authority_service.rs

```diff
@@ -389,15 +397,12 @@ impl<C: CoreThreadDispatcher> NetworkService for AuthorityService<C> {
                 .inc_by(missing_excluded_ancestors.len() as u64);
 
             let synchronizer = self.synchronizer.clone();
-            tokio::spawn(async move {
-                // schedule the fetching of them from this peer in the background
+            spawn_monitored_task!(async move {
                 if let Err(err) = synchronizer
                     .fetch_blocks(missing_excluded_ancestors, peer)
                     .await
                 {
-                    warn!(
-                            "Errored while trying to fetch missing excluded ancestors via synchronizer: {err}"
-                        );
+                    debug!("Failed to fetch missing soft links via synchronizer: {err}");
```

Comment:
IMO if you are changing verbage here to soft links it would be best to change it everywhere excluded ancestors is used for consistency in logging. Also similarily to the above comment, would be good to add something to the effect of this is a best effort fetch to improve block propagation and can be fetched later if needed for any validators looking at this log line in the future

Reply:
Switching back to `excluded ancestors`. This doesn't get retried by periodic fetching though.